pom.xml

```diff
@@ -0,0 +1,75 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.github.wechaty</groupId>
+  <artifactId>wechaty-grpc-proto</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0.0</version>
```

Comment:
I believe this version number is the module number of our Java GRPC.

It should be sync to the exact version in our `package.json`.

I'm not sure the workflow between the GitHub and the Marven Central Repo, it will be great if this can be archived via the DevOps workflow (GitHub Actions in our case)

Reply:
fix too